docs/topics/con_about-shared-messaging-work-queue.adoc

```diff
@@ -0,0 +1,14 @@
+[id='about-messaging-work-queue_{context}']
```

Comment:
Difference between `docs/topics/con_about-messaging-work-queue.adoc` and `docs/topics/con_about-shared-messaging-work-queue.adoc`?

Reply:
@Ladicek Given that they're identical incontent, save of the name, the reduplication is likely the result of, removed `docs/topics/con_about-shared-messaging-work-queue.adoc` and retained the other one. Thank you:)